From 60557194546f89a76eff4f9615e97b9335319853 Mon Sep 17 00:00:00 2001
From: Xu Xuehui <xxh@rock-chips.com>
Date: Tue, 26 Dec 2017 15:07:30 +0800
Subject: [PATCH] Revert "net: wireless: rockchip_wlan: ap6255: fix irq
 abnormal."

This reverts commit a7e71a03cf1fe23ce97bf07fb6143431f4abb8fa

Change-Id: I50ef1401a18c71acaddaf8dbda1c525d71ba4392
Signed-off-by: Xu Xuehui <xxh@rock-chips.com>
---
 .../net/wireless/rockchip_wlan/rkwifi/bcmdhd/bcmsdh_linux.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/bcmsdh_linux.c b/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/bcmsdh_linux.c
index 5231c7739a75..9e3f641bf038 100644
--- a/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/bcmsdh_linux.c
+++ b/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/bcmsdh_linux.c
@@ -51,9 +51,7 @@ extern void dhdsdio_isr(void * args);
 #include <linux/platform_data/gpio-odin.h>
 #endif /* defined(CONFIG_ARCH_ODIN) */
 #include <dhd_linux.h>
-#include <linux/rfkill-wlan.h>
 
-extern int get_wifi_chip_type(void);
 
 /* driver info, initialized when bcmsdh_register is called */
 static bcmsdh_driver_t drvinfo = {NULL, NULL, NULL, NULL};
@@ -359,7 +357,6 @@ int bcmsdh_oob_intr_register(bcmsdh_info_t *bcmsdh, bcmsdh_cb_fn_t oob_irq_handl
 	void* oob_irq_handler_context)
 {
 	int err = 0;
-	int type;
 	bcmsdh_os_info_t *bcmsdh_osinfo = bcmsdh->os_cxt;
 
 	if (bcmsdh_osinfo->oob_irq_registered) {
@@ -391,9 +388,6 @@ int bcmsdh_oob_intr_register(bcmsdh_info_t *bcmsdh, bcmsdh_cb_fn_t oob_irq_handl
 		return err;
 	}
 
-	type = get_wifi_chip_type();
-	if (type == WIFI_AP6255)
-		bcmsdh_oob_intr_set(bcmsdh, FALSE);
 #if defined(DISABLE_WOWLAN)
 	SDLX_MSG(("%s: disable_irq_wake\n", __FUNCTION__));
 	bcmsdh_osinfo->oob_irq_wake_enabled = FALSE;
-- 
2.35.3

